docs/quickstarts.rst

```diff
@@ -29,26 +29,36 @@ Once generated, run ``cp .env_template .env`` in the fv3net root and edit the ``
 Cloud Authentication
 --------------------
 
-The ``fv3net`` project currently utilizes Google Cloud to deploy workflow items to services such as Kubernetes and Dataflow.  Authentication requires an installation of `Google Cloud SDK <https://cloud.google.com/sdk/docs/install>`_.
+The ``fv3net`` project currently utilizes Google Cloud to deploy workflow items to services such as Kubernetes and Dataflow.
+Authentication requires an installation of `Google Cloud SDK <https://cloud.google.com/sdk/docs/install>`_.
 
-Authentication obtained via ``gcloud auth login`` does not work well with secrets management and is not used by many APIs. Service account key-based authentication works much better, because the service account key is a single file that can be deployed in a variety of contexts (K8s cluter, VM, etc) (`Google docs on service accounts <https://cloud.google.com/iam/docs/service-accounts>`_). Many Python APIs can authenticate with google using the ``GOOGLE_APPLICATION_CREDENTIALS`` environmental variable `(See Google authentication details) <https://cloud.google.com/sdk/docs/authorizing>`_.
-
-* If gcloud is a fresh install, initialize and grab a keyfile::
+* If gcloud is a fresh install,
 
     > gcloud init
     > gcloud auth login
+
+Authentication obtained via ``gcloud auth login`` does not work well with secrets management and is not used by many APIs.
+Depending on whether you are working on your local machine or on a VM.
+
+* Local machine
+
+    > gcloud auth application-default login
+
+    If you are working in a docker container, you can bind mount in the necessary credentials location in your `docker-compose.yaml`::
+
+      volumes:
+        - ~/.config/gcloud:/root/.config/gcloud
+
+* VM
+
```

Comment:
I would just remove the service account key information for now and say something along the lines of.

```
VMs are currently automatically provisioned with an active service account specific to the VM user.  No 
further action should be required, but we are looking into also providing a key file referenced by
GOOGLE_APPLICATION_CREDENTIALS=... to simplify integration with many tools that use the environment
variable API.
```